src/Internal/SetupDecryptionHandler.cs

```diff
@@ -373,6 +390,11 @@ protected void DecryptObject(byte[] decryptedEnvelopeKeyKMS, GetObjectResponse g
                         var instructionFileRequest = EncryptionUtils.GetInstructionFileRequest(getObjectResponse, EncryptionUtils.EncryptionInstructionFileSuffix);
                         instructionFileResponse = GetInstructionFile(instructionFileRequest);
                     }
+                    catch (AmazonS3Exception amazonS3ExceptionInner) when (amazonS3ExceptionInner.ErrorCode == EncryptionUtils.NoSuchKey)
+                    {
+                        throw new AmazonServiceException($"Exception encountered while fetching Instruction File. Ensure the object you are" +
+                            $" attempting to decrypt has been encrypted using the S3 Encryption Client and instruction files are enabled.", amazonS3ExceptionInner);
```

Comment:
Same comment as above. Remove the `and instruction files are enabled`.

Reply:
Done